mlflow/tracing/fluent.py

```diff
@@ -79,6 +83,10 @@ def my_function(x, y):
     manager, but requires less boilerplate code.
 
     .. code-block:: python
+        :test:
+
+        import mlflow
+
```

Comment:
We currently don't any policy on what should be tested, but all basic examples should be runnable.

Reply:
This also prevents issues report like "hey, this example is broken".